emgapi/management/commands/populate_metagenomics_exchange.py

```diff
@@ -131,6 +131,9 @@ def process_to_index_and_update_records(self, analyses_to_index_and_update):
                         mgya=annotation_job.accession,
                         sequence_accession=sequence_accession,
                     )
+                    if not response:
+                        logging.info(f"Error occured {annotation_job}")
```

Comment:
```suggestion
                        logging.warning(f"Error occurred {annotation_job}")
```
I suppose it is bad enough for a warning :)